packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx

```diff
@@ -1,33 +1,64 @@
 import React from 'react';
 
+import ApolloClient, {ApolloClientOptions} from 'apollo-client';
 import {useSerialized} from '@shopify/react-html';
-import {ApolloProvider, createSsrExtractableLink} from '@shopify/react-graphql';
+import {
+  ApolloProvider,
+  createSsrExtractableLink,
+  createOperationDetailsLink,
+  GraphQLOperationDetails,
+} from '@shopify/react-graphql';
 import {useLazyRef} from '@shopify/react-hooks';
 import {createApolloBridge} from '@shopify/react-effect-apollo';
+import {useNetworkManager} from '@shopify/react-network';
+
+export type GraphQLClientOptions = ApolloClientOptions<any>;
+export const GRAPHQL_OPERATIONS = Symbol('graphQLOperations');
 
 interface Props {
   children?: React.ReactNode;
-  createClient(): import('apollo-client').ApolloClient<any>;
+  createClientOptions(): ApolloClientOptions<any>;
```

Comment:
It's to allow users to pass in functions to customize the returned options - [context](https://github.com/Shopify/quilt/pull/1005#discussion_r325369735).

Reply:
> for the same reasons the previous api was createClient(): ApolloClient not client: ApolloClient.

😬 I dont know if I entirely understand why. I can only guess it is because of link or resolvers? Everything else seems pretty static.